### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by ElJaeger** [syssi/homeassistant-goecharger-mqtt#146] [Ideas]
A feature for different languages would be great. I've already taken a look, but it seems friendly names are hardcoded in the corresponding definitions. Would it be possible to create a key-based naming, so we can add other languages? I would like to volunteer for german!

**Comment by syssi** [comment]
Note to myself: There is a translation layer which can be used to translate entity states

https://developers.home-assistant.io/docs/internationalization/core/

**Reply by syssi** [reply]
This is an code example: https://github.com/home-assistant/core/pull/122367/files

https://developers.home-assistant.io/blog/2024/01/19/entity-translations-placeholders
